front/app/components/HookForm/Select/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  Select as SelectComponent,
+  SelectProps,
+} from '@citizenlab/cl2-component-library';
+import Error, { TFieldName } from 'components/UI/Error';
+import { Controller, useFormContext } from 'react-hook-form';
+import { CLError } from 'typings';
+interface Props extends Omit<SelectProps, 'onChange'> {
+  name: string;
+}
+
+const Select = ({ name, ...rest }: Props) => {
+  const {
+    setValue,
+    formState: { errors },
+    control,
+  } = useFormContext();
+
+  const validationError = errors[name]?.message as string | undefined;
+
+  const apiError =
+    (errors[name]?.error as string | undefined) &&
+    ([errors[name]] as unknown as CLError[]);
+
+  return (
+    <>
+      <Controller
+        name={name}
+        control={control}
+        render={({ field: { ref: _ref, ...field } }) => {
+          return (
+            <SelectComponent
+              id={name}
+              {...field}
+              {...rest}
+              onChange={(e) => setValue(name, e.value)}
+            />
+          );
+        }}
+      />
+      {validationError && (
+        <Error
+          marginTop="8px"
+          marginBottom="8px"
```

Comment:
Minor. I noticed a number of places have the same marginTop and marginBottom. Probably a good consideration for `my` 🤔 . Out of scope for this PR but I thought I'd point it out so we could probably consider it down the line.

Or maybe we could create a styled Error version used in hook forms that has similar properties. Some of them look similar but I might be mistaken

Reply:
Indeed - the issue is that the error component is not taking advantage of our `BoxMarginProps` yet - definitely something to improve in the future, though 👍